docs/glsl-fuzz-walkthrough.md

```diff
@@ -544,31 +538,47 @@ in the next section,
 we will reduce the shader to obtain a smaller and simpler shader
 that is more useful in understanding the root cause of the bug.
 
+Return to the results table view
+and click on the image under `variant_009`
+to reveal the single result page.
 
-## Queuing a bad image reduction
+![Variant_009 single result](images/screenshot-single-result-bad-image.png)
 
-> Reductions for bad results from compute shaders cannot be launched via the Web UI.  [See how to launch them via the command line](#Performing-a-bad-result-reduction-for-a-compute-shader).
+*Images sizes have been reduced to fit more information into the screenshot.*
 
-Return to the results table view:
+As seen above,
+the reference image and result image are shown for comparison,
+as well as the results of the individual image comparison metrics
+(if the images are different).
+The *histogram distance* is shown, which can be useful when
+launching a reduction, as explained below. 
 
-![Table of image results](images/screenshot-results-table.png)
 
-Click on the image under `variant_001`
-to reveal the single result page,
-and click the "Reduce result" button
-to reveal the reduction panel:
+## Queuing a bad image reduction
 
-![Variant_001 single result](images/screenshot-single-result-bad-image.png)
+> Reductions for bad results from compute shaders cannot be launched via the Web UI.  [See how to launch them via the command line](#Performing-a-bad-result-reduction-for-a-compute-shader).
+
+![Variant_009 single result reduction panel](images/screenshot-single-result-bad-image-reduction.png)
 
+From the single result page,
+click the "Reduce Result" button to reveal the reduction panel,
+as shown in the above screenshot.
 From here,
 we can queue a reduction of the variant shader
 to find a smaller, simpler shader
 that still exposes the bug.
-The default reduction settings are sufficient, so just click
+The "fuzzy diff" algorithm is selected by default,
+which will work in this case,
+as both metrics detected the image as different from the reference.
+You can select the "HISTOGRAM_CHISQR" metric from the "Comparison metric" dropdown
+to switch the histogram comparison metric;
+you can then adjust the threshold based on the histogram distance reported
+higher up the page, although we find that 100.0 usually works,
+which is the default.
+In this case,
```

Comment:
Saying "In this case, the ..." sounds like they'll actually be reducing this bug, but they won't as it's just a bug we found.